src/Connection.ts

```diff
@@ -1735,12 +1735,12 @@ export class Connection<
 
 	/**
 	 * Write a file of an adapter.
-	 * @param adapter The adapter name.
-	 * @param fileName The file name.
+	 * @param {string} namespace (this may be the adapter name, the instance name or the name of a storage object within the adapter).
+	 * @param {string} fileName The file name.
```

Comment:
```suggestion
	 * @param namespace (this may be the adapter name, the instance name or the name of a storage object within the adapter).
	 * @param fileName The file name.
```